web/blog/2024-09-24-why-your-emails-arent-getting-delivered-and-how-to-fix-this-problem.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Why Your Emails Aren't Getting Delivered and How to Fix This Problem"
+authors: [milica]
+image: /img/email/email-cover.jpg
+tags: [webdev, tips, emails]
+---
+import ReactPlayer from 'react-player'
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ImgWithCaption from './components/ImgWithCaption'
+
+If you’re new to [Wasp](https://wasp-lang.dev/), and you’ve just built and deployed your app, it’s possible that your emails aren’t reaching your users. Don’t be too alarmed. This is a common issue that affects many developers who have newly registered domains.
+
+In other words, your users and their email servers pulled a Gandalf move on you. No worries, this is something you can fix!
+
+![https://media2.giphy.com/media/WpD30tFjzosDn7amXq/giphy.gif?cid=7941fdc68jh5kd0zboo37lcp48knhd8bewokjgz7kc9juwov&ep=v1_gifs_search&rid=giphy.gif&ct=g](https://media2.giphy.com/media/WpD30tFjzosDn7amXq/giphy.gif?cid=7941fdc68jh5kd0zboo37lcp48knhd8bewokjgz7kc9juwov&ep=v1_gifs_search&rid=giphy.gif&ct=g)
```

Comment:
FYI it would be better to put some description inside of `[]` to make our blog more accessible for screen readers.

Reply:
Also, let's check if Giphy allows direct linking to gifs - if not, download them and link them locally :)

Reply:
Oh yeah, they actually encourage embedding :) 

If you take it away from their platform it counts as taking it away from them. 

The Gandalf one is owned by HBO for example :)

Reply:
Good to know!